example/example.dart

```diff
@@ -5,6 +5,11 @@
 import 'package:web/web.dart';
 
 void main() {
-  final div = document.querySelector('div') as HTMLDivElement;
+  // ignore: unused_local_variable
+  final HtmlDocument foo = document;
+
+  final doc = document as HtmlDocument;
```

Comment:
This location does _not_ offer a quick fix. Should I define the fix data differently? Is the rename fix expected to cover this case?

cc @keertip

Reply:
I would have expected the rename change to cover this case. It looks like a bug to me.

Reply:
As Brian said, it should, otherwise it is a bug. Could you open an issue to capture this?

Reply:
https://github.com/dart-lang/sdk/issues/54617